addons/common/functions/fnc_findUnloadPosition.sqf

```diff
@@ -40,12 +40,22 @@ if (_checkVehicleIsStable) then {
 };
 
 private _radiusOfItem = 1;
-if (_typeOfCargo isKindOf "CAManBase") then {
+if (_cargo isKindOf "CAManBase") then {
     _radiusOfItem = 1.1;
 } else {
     //`sizeOf` is unreliable, and does not work with object types that don't exist on map, so estimate size based on cargo size
-    if (isNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) then {
-        _radiusOfItem = (((getNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) ^ 0.35) max 0.75);
+    private _typeOfCargo = if (_cargo isEqualType "") then {_cargo} else {typeOf _cargo};
+    private _itemSize = if (isNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) then {
+        getNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size));
+    } else {
+        if (["ace_cargo"] call EFUNC(common,isModLoaded)) then {
```

Comment:
`EFUNC(common,isModLoaded)` -> `FUNC(isModLoaded)`

Reply:
`EFUNC(common,isModLoaded)` -> ace_common_fnc_isModload
`FUNC(isModLoaded)` -> ace_cargo_fnc_isModload

Because the second doesn't exist I am using the first one as suggested by @jonpas  here: https://github.com/acemod/ACE3/pull/6364#discussion_r190597398

Reply:
here we are in `common` module (`addons/common/functions/fnc_findUnloadPosition`) so FUNC is `ace_common_fnc`

Reply:
woups!!!